src/utils/locale.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * Mainly copied from Snooty locale.js and other snooty utils
+ * Nov 14, 2023
+ */
+
+/**
+ * Key used to access browser storage for user's preferred locale
+ */
+export const STORAGE_KEY_PREF_LOCALE = 'preferredLocale';
+
+// Update this as more languages are introduced
+// Because the client-side redirect script cannot use an import, PLEASE remember to update the list of supported languages
+// in redirect-based-on-lang.js
```

Comment:
(Nit) To avoid confusion about this file in this repo, it might be fine to exclude this comment since we don't currently have the client-side redirect based on language? **However**, I can see it being fine to keep in case we plan on introducing the client-side redirect logic soon (i.e. before we move off of Redoc)